### PR TITLE
chore(storybook): Upgrade to version 6 - FRONT-1483

### PR DESCRIPTION
**Pull request by planctus**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)
- [ ] The redirects in `src/website/src/routes/Redirects.jsx` are up-to-date

### CODE REVIEW

File: src/systems/ec/implementations/react/storybook/.storybook/manager.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ const theme = create({
 });
 
 addons.setConfig({
+  showRoots: false,
```
Comment (kalinchernev) [inline src/systems/ec/implementations/react/storybook/.storybook/manager.js]: We are currently using the default which is true. Why changing?

File: src/systems/ec/implementations/react/storybook/.storybook/main.js [hunk 2]
```diff
@@ -10,26 +10,35 @@ const stories = [
...
     path.resolve(__dirname, '../../../../../..')
   );
 
-  // Exclude node_modules
-  baseConfig.module.rules[0].exclude = /node_modules/;
+  defaultConfig.module.rules[0].exclude = /node_modules/;
+
+  // Change media dist folder
+  defaultConfig.module.rules[5].query.name =
+    'dist/images/[name].[hash:8].[ext]';
```
Comment (kalinchernev) [inline src/systems/ec/implementations/react/storybook/.storybook/main.js:30]: I am not sure what's the idea behind this change

**Review by kalinchernev — COMMENTED** [review]
Thanks for the massive amount of updates here!
I've added 2 items which didn't quite fit during the review. In overall, works fine locally and in the preview

File: src/systems/ec/implementations/react/storybook/.storybook/manager.js (hunk 1 above)
Comment (planctus) [reply src/systems/ec/implementations/react/storybook/.storybook/manager.js]: Ah, this is a mistake, yeah, i moved this according to the docs for storybook 6 but i changed it's value inadvertently, it was set to true.

File: src/systems/ec/implementations/react/storybook/.storybook/main.js (hunk 2 above)
Comment (planctus) [reply src/systems/ec/implementations/react/storybook/.storybook/main.js:30]: it was already in place, just that the rule position in the array changed.

**Pull request merged**